.github/workflows/release.yml

```diff
@@ -19,12 +19,22 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: "3.10"
-
+      
+      - name: Prepare Package Version
+        run: |
+          sed -i 's/__PACKAGE_VERSION__/1.34.0/g' zanshincli/version.py
```

Comment:
This is incorrect, we shouldn't hard code the version. The previous code obtained the version to be used from the Github tag of the release, and we should continue doing so.